semcore/data-table/src/components/Body/Body.tsx

```diff
@@ -70,7 +70,9 @@ class BodyRoot<UniqKeyType> extends Component<DataTableBodyProps<UniqKeyType>, {
         if (previousRowKey) {
           const previousRow = this.asProps.flatRows.find((r) => r[UNIQ_ROW_KEY] === previousRowKey);
           if (previousRow) {
-            this.closeAccordion(previousRow, closeDuration);
+            setTimeout(() => {
+              this.closeAccordion(previousRow, closeDuration);
+            }, openDuration / 2);
```

Comment:
```suggestion
            }, openDuration / 3);
```
can we decrease the timeout a bit? it feels a bit slow, I tried it with `/ 3` - it feels better